pkgs/applications/audio/squeezelite/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchgit, alsaLib, flac, libmad, libvorbis, faad2, mpg123 }:
+
+stdenv.mkDerivation {
+  name = "squeezelite-git-2016-05-27";
+
+  src = fetchgit {
+    url = "https://github.com/ralph-irving/squeezelite.git";
+    rev = "e37ed17fed9e11a7346cbe9f1e1deeccc051f42e";
+    sha256 = "15ihx2dbp4kr6k6r50g9q5npqad5zyv8nqf5cr37bhg964syvbdm";
+  };
+
+  buildInputs = [ stdenv alsaLib flac libmad libvorbis faad2 mpg123 ];
```

Comment:
I started from a nix-shell definition and forgot to remove it when making the package. Thanks.